cmd/seed/organization.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Let's have at least 2 organizations and 2 projects? Useful for seeing how a list looks in our UI. Also I don't see any policies being created to assign role to users.

Reply:
done